src/pages/ProductDetail/ProductDetail.js

```diff
@@ -1,9 +1,270 @@
 import React, { Component } from 'react';
+import ProductPhoto from './ProductPhoto';
+import ProductSize from './ProductSize';
+import ProductBonus from './ProductBonus';
+import ShoppingBag from './ShoppingBag';
 import './ProductDetail.scss';
 
 class ProductDetail extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      productInfoList: [],
+      imageNum: 0,
+      isSize: true,
+      isBonus: true,
+      isShoppingBag: true,
+      size: '',
+    };
+  }
+
+  componentDidMount() {
+    fetch('/data/productData.json')
+      .then(res => res.json())
+      .then(data => {
+        this.setState({
+          productInfoList: data.results,
+        });
+      });
+  }
+
+  postProduct = () => {
+    const { productInfoList } = this.state;
+    fetch('http://10.58.7.79:8000/users/signup', {
+      method: 'POST',
+      body: JSON.stringify({
+        product_id: productInfoList.product_id,
+      }),
+    });
+  };
+
+  changeSize = size => {
+    this.setState({ size: size }, this.hideSizePick());
+  };
+
+  hideSizePick = () => {
```

Comment:
```suggestion
  toggleSizePick = () => {
```
상태를 뒤집는 동작이기 때문에 toggle이라는 명칭이 더 부합해보입니다

Reply:
넵